examples/chip-tool/commands/common/CHIPCommand.cpp

```diff
@@ -77,6 +81,16 @@ CHIP_ERROR GetAttestationTrustStore(const char * paaTrustStorePath, const chip::
     *trustStore = &attestationTrustStore;
     return CHIP_NO_ERROR;
 }
+
+using ::chip::Tracing::ScopedRegistration;
+using ::chip::Tracing::LogJson::LogJsonBackend;
+
+LogJsonBackend log_json_backend;
```

Comment:
Consider adding to Linux sample apps as well (see Options.h/Options.cpp and `--trace_decode 1` handling handling in the main bodies. This will give the abiltiie to trace to all-clusters app as well. Can be done in a follow-up.

Reply:
This is intended as a followup after I have data handling. Having all handling in one PR is what I want to avoid (too large of a change, too likely to have conflicts).

Will happen for sure though.